### PR TITLE
chore: Apply Black code formatter

### PR DESCRIPTION
**Pull request by ki4070ma**
### Background

We don't need to pay attention to code format anymore.

Please see detailed concept here.
https://github.com/psf/black

Any comments are welcome.

### Tests
- [x] Functional tests passed (except for keyboard)
   * https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=13584&view=ms.vss-test-web.build-test-results-tab
- [x] Travis CI including black code formatter check  passed

### CODE REVIEW

File: appium/webdriver/appium_connection.py [hunk 1]
```diff
@@ -24,14 +24,19 @@
...
+        headers = RemoteConnection.get_remote_connection_headers(
+            parsed_url, keep_alive=keep_alive
+        )
+        headers["User-Agent"] = "appium/python {} ({})".format(
+            library_version(), headers["User-Agent"]
+        )
+        if parsed_url.path.endswith("/session"):
             # https://github.com/appium/appium-base-driver/pull/400
-            headers['X-Idempotency-Key'] = str(uuid.uuid4())
+            headers["X-Idempotency-Key"] = str(uuid.uuid4())
```
Comment (mykola-mokhnach) [inline appium/webdriver/appium_connection.py:40]: I prefer single quotes everywhere

File: appium/webdriver/errorhandler.py [hunk 2]
```diff
@@ -25,7 +25,9 @@ def check_response(self, response: Dict) -> None:
         try:
             super().check_response(response)
         except WebDriverException as wde:
-            if wde.msg == 'No such context found.':
-                raise NoSuchContextException(wde.msg, wde.screen, wde.stacktrace) from wde
+            if wde.msg == "No such context found.":
+                raise NoSuchContextException(
```
Comment (mykola-mokhnach) [inline appium/webdriver/errorhandler.py:29]: It is ok to have strings up to 120 chars in length

File: setup.cfg [hunk 3]
```diff
@@ -1,10 +1,12 @@
-[pycodestyle]
-max-line-length = 120
-
 [isort]
-multi_line_output = 3
 known_third_party = dateutil,httpretty,pytest,selenium,setuptools,urllib3,mock,sauceclient
 known_first_party = test,appium
+multi_line_output = 3
```
Comment (ki4070ma) [inline setup.cfg:4]: From https://black.readthedocs.io/en/stable/compatible_configs.html#isort

**Comment by mykola-mokhnach** [comment]
I like how it formats the code now

**Review by KazuCocoa — APPROVED** [review]
like it :)

**Pull request merged**